.circleci/config.yml

```
@@ -285,3 +312,14 @@ workflows:
           filters: *default-filters
           requires:
             - build
+      - build-docusaurus-website:
+          filters: *default-filters
+          requires:
+            - build
+      - deploy-docusaurus-website:
+          filters:
+            <<: *default-filters
+            branches:
+              only: master
```

```suggestion
          filters:
            branches:
              only: master
```

since we're using `only master` condition I think you can remove "default filters"